examples/nlu-assistant/README.md

```diff
@@ -0,0 +1,507 @@
+  await nlu.createModel({
+    template: ModelTemplatesType.SIMPLE_NN,
+    learningRate: 5e-3,
+    trainableEmbeddings: true,
+  })
+  await nlu.train(xTrain, yTrain, { epochs: 8 })
+  await nlu.saveModel()
+  console.log('Accuracy:', nlu.evaluate(xTest, yTest))
+  console.log('Model saved.')
+})()
+```
+
+Let's break it down into little pieces:
+
+1. First of all, you need to define which is the target language to be trained. You must specify both the language and the directory from where the utterances will be retrieved:
+   ```ts
+   const LANGUAGE = 'en'
+   const UTTERANCES_PATH = join(
+     process.cwd(),
+     'src',
+     'nlu',
+     'utterances',
+     LANGUAGE
+   )
+   ```
+2. Now we will initialize an instance of `BotonicNLU` and feed it with our data:
+   ```ts
+   const nlu = new BotonicNLU()
+   const data = nlu.readData({
+     path: UTTERANCES_PATH,
+     language: LANGUAGE,
+     maxSeqLen: 20,
+   })
+   ```
+   `maxSeqLen` defines the maximum length of the sentences to train the NN. As webchat is a messaging channel, a `maxSeqLen` of 20 will be enough to cover our needs (_it's pretty uncommon that a user types an input of more of 20 words_).
+3. Before start training our model, we need to split the data into two sets; one for training and one for evaluating the model. `testPercentage` is the percentage of examples that will be used to do the evaluation. This can be done with:
+   ```ts
+   const [xTrain, xTest, yTrain, yTest] = nlu.trainTestSplit({
+     data: data,
+     testPercentage: 0.1,
+   })
+   ```
+4. Finally, we will use a predefined template of NN to do the training:
+
+   ```ts
+   ;(async () => {
+     await nlu.createModel({
+       template: ModelTemplatesType.SIMPLE_NN,
+       learningRate: 5e-3,
+       trainableEmbeddings: true,
+     })
+     await nlu.train(xTrain, yTrain, { epochs: 8 })
+     await nlu.saveModel()
+     console.log('Accuracy:', nlu.evaluate(xTest, yTest))
+     console.log('Model saved.')
+   })()
+   ```
+
+   We won't go over the details a this moment, but it's important you notice that the main logic here consists of: specifying a model, training it during some epochs and finally save and evaluate the results.
+
+5. Now run `botonic train`.
+
+   **Output:**
+
+   ```
+   ...
+   3283ms 15198us/step - acc=1.00 loss=8.80e-4 val_acc=0.167 val_loss=6.70
+   Accuracy: 0.6333333333333333
+   Model saved.
+   ```
+
+   As you can see, the `acc` (training accuracy) is high, and `val_acc` (validation accuracy) is low. This means that our network did a good job classifying the intents that it already knows, but it's performance when seeing new examples is really poor.
+   We could try to do lots of changes within this code, but we will never obtain significantly better results. To address this problem we will need to create a "dummier" model to be more flexible in terms of learning and correct this issue (known as `overfitting`), so we will need to make use of a custom NLU model.
```

Comment:
I would not downgrade our SimpleNN template, instead, I would use this opportunity to present the other way of implementing a NN: **Creating your own model**:

>To address this problem we are going to try the other way of implementing a model. This option offers the possibility of creating a custom model without using any template.